src/main/java/com/javalearn/circle/Director.java

```diff
@@ -6,36 +6,34 @@
 import com.javalearn.circle.exceptions.DataException;
 import com.javalearn.circle.logic.CircleCreator;
 import com.javalearn.circle.logic.CircleValidator;
+import org.apache.log4j.Logger;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class Director {
 
-    private DataReader reader;// = new SimpleDataReader();
-    private CircleValidator validator;// = new CircleValidator();
-    private CircleCreator creator;// = new CircleCreator();
+    private DataReader reader;
+    private CircleValidator validator;
+    private CircleCreator creator;
 
+    private final static Logger LOGGER = Logger.getLogger(SimpleDataReader.class);
 
     public Director (DataReader reader, CircleValidator validator, CircleCreator creator){
         this.reader = reader;
         this.validator = validator;
         this.creator = creator;
     }
 
-/*
-    Director (){
-        this.reader = new SimpleDataReader();
-        this.validator = new CircleValidator();
-        this.creator = new CircleCreator();
-    }
-
- */
-
-
     public List<Circle> readAndCreateCircles (String path) throws DataException {
         List<Circle> circles = new ArrayList<>();
-        List<String> lines = reader.read(path);
+        List<String> lines = new ArrayList<>();
+
+        try {
+            lines = reader.read(path);
+        } catch (DataException e){
+            LOGGER.error (e.getMessage(), e);
+        }
 
         for (String line: lines){
```

Comment:
put this inside try, because if there is error in reading data no sense to continue processing